cmd/workflow.go

```diff
@@ -1,40 +1,145 @@
 package cmd
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/spf13/cobra"
 
 	e "github.com/cloudposse/atmos/internal/exec"
 	"github.com/cloudposse/atmos/pkg/schema"
+	"github.com/cloudposse/atmos/pkg/ui/markdown"
 	u "github.com/cloudposse/atmos/pkg/utils"
 )
 
+// ErrorMessage represents a structured error message
+type ErrorMessage struct {
+	Title      string
+	Details    string
+	Suggestion string
+}
+
+// String returns the markdown representation of the error message
+func (e ErrorMessage) String() string {
+	var sb strings.Builder
+	sb.WriteString(fmt.Sprintf("# ❌ %s\n\n", e.Title))
+
+	if e.Details != "" {
+		sb.WriteString(fmt.Sprintf("## Details\n%s\n\n", e.Details))
+	}
+
+	if e.Suggestion != "" {
+		sb.WriteString(fmt.Sprintf("## Suggestion\n%s\n\n", e.Suggestion))
+	}
+
+	return sb.String()
+}
+
+// renderError renders an error message using the markdown renderer
+func renderError(msg ErrorMessage) error {
+	renderer, err := markdown.NewRenderer(
+		markdown.WithWidth(80),
+	)
+	if err != nil {
+		return fmt.Errorf("failed to create markdown renderer: %w", err)
+	}
+
+	rendered, err := renderer.Render(msg.String())
+	if err != nil {
+		return fmt.Errorf("failed to render error message: %w", err)
+	}
+
+	fmt.Print(rendered)
+	return nil
+}
+
 // workflowCmd executes a workflow
 var workflowCmd = &cobra.Command{
 	Use:   "workflow",
 	Short: "Execute a workflow",
-	Long:  `This command executes a workflow: atmos workflow <name> -f <file>`,
+	Long:  `This command executes a workflow: atmos workflow <name> --file <file>`,
 	Example: "atmos workflow\n" +
-		"atmos workflow <name> -f <file>\n" +
-		"atmos workflow <name> -f <file> -s <stack>\n" +
-		"atmos workflow <name> -f <file> --from-step <step-name>\n\n" +
+		"atmos workflow <name> --file <file>\n" +
+		"atmos workflow <name> --file <file> --stack <stack>\n" +
+		"atmos workflow <name> --file <file> --from-step <step-name>\n\n" +
```

Comment:
```suggestion
		"- Run the workflow called `<name>` in the `<file>`\n  ```shell\n  atmos workflow <name> --file <file>\n  ```\n" +
		"- Run the workflow called `<name>` in the `<file>` with the `<stack>` context.\n. ```shell\n  atmos workflow <name> --file <file> --stack <stack>\n  ```\n" +
		"- Run the `<name>` workflow in the `<file>` starting from `<step-name>`\n  ```shell\n  atmos workflow <name> --file <file> --from-step <step-name>\n  ```\n" +
```